jib-core/src/main/java/com/google/cloud/tools/jib/api/DockerDaemonImage.java

```diff
@@ -49,7 +49,7 @@ public static DockerDaemonImage named(String imageReference)
   }
 
   private final ImageReference imageReference;
-  private Path dockerExecutable = CliDockerClient.DEFAULT_DOCKER_CLIENT;
+  private Path dockerExecutable = CliDockerClient.getExistingDefaultDocker();
```

Comment:
What will happen if a custom path to a Docker executable is set and jib identifies a Podman default installation?
I can see that this variable is also used by the CliDockerClient for command issuance.
I guess the command has to be deducted from the custom path to a container engine as well.

Reply:
If custom path is set it will be used, this it default path to executable if none is provided.